src/systemctl/systemctl.c

```diff
@@ -5561,6 +5562,7 @@ static int show_one(
                 { "NConnections",                   "u",               NULL,           offsetof(UnitStatusInfo, n_connections)                     },
                 { "NRefused",                       "u",               NULL,           offsetof(UnitStatusInfo, n_refused)                         },
                 { "Accept",                         "b",               NULL,           offsetof(UnitStatusInfo, accept)                            },
+                { "Flush",                          "b",               NULL,           offsetof(UnitStatusInfo, flush)                             },
```

Comment:
this can be dropped, if you don't actually intend to show the value in "systemctl status" (and I don't think we should show it)